src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs

```diff
@@ -147,5 +148,20 @@ internal bool HasClientCredentials
         /// which is the value used by Microsoft.Identity.Web.UI.
         /// </summary>
         public PathString ErrorPath { get; set; } = new PathString("/MicrosoftIdentity/Account/Error");
+
+        /// <summary>
+        /// Delegates enabling the application to provide a client assertion instead of
+        /// <see cref="ClientCertificates"/> or client secrets.
+        /// </summary>
+        /// <example>
+        /// In Startup.cs, after AddMicrosoftIdentityWebXX, add another instruction:
+        /// <code>
+        /// services.ConfigureOptions&lt;MicrosoftIdentityOptions&gt;(OpenIdConnectDefaults.AuthenticationScheme,
+        /// options => {
+        ///   options.ClientAssertionDescription = GetClientAssertionProvider();
+        /// });
+        /// </code>
+        /// </example>
+        public ClientAssertionDescription? ClientAssertionDescription { get; set; }
```

Comment:
nit: i would move this section to below the part on certs

Reply:
yes. I'll move it. I was trying to not change the order of the fields in the data structure to have a binary compatibility of the data structure. Old outdated reflexes of a time where C was ruling the world.